epick_driver/src/default_driver.cpp

```diff
@@ -140,35 +140,33 @@ void DefaultDriver::disconnect()
 
 void DefaultDriver::activate()
 {
-  const uint8_t max_absolute_pressure =
-      static_cast<uint8_t>(std::clamp(std::round(max_vacuum_pressure_ + kAtmosphericPressure), 0.0f, 255.0f));
+  const uint8_t grip_max_absolute_pressure = static_cast<uint8_t>(kAtmosphericPressure);
```

Comment:
What's the point of static_cast<uint8_t> here?

Reply:
The  pressure, which is given a float, must be converted into a byte before being sent to the driver.

Reply:
Explained offline.